toqito/channel_ops/kraus_to_channel.py

```diff
@@ -0,0 +1,54 @@
+"""Converts Kraus operators into the corressponding quantum channel (i.e. superoperator)."""
+
+import numpy as np
+
+from toqito.matrix_ops import tensor
+
+
+def kraus_to_channel(
+    kraus_list: list[tuple[np.ndarray, np.ndarray]]
+) -> np.ndarray:
+    r"""Convert a collection of Kraus operators into the corresponding quantum channel (superoperator).
+
+    (Section: Kraus Representations of :cite:`Watrous_2018_TQI`).
+
+    This function computes the superoperator representation of a quantum channel from its Kraus representation.
+    Given a list of Kraus operators \(\{A_i, B_i\}\), the superoperator \(\mathcal{E}\) is computed as:
+
+    \[
+    \mathcal{E}(\rho) = \sum_i B_i \rho A_i^\dagger
+    \]
+
+    The resulting quantum channel can be applied to density matrices by reshaping them into vectorized form.
+
+    Examples
+    ========
+
+    Constructing a simple quantum channel from Kraus operators:
+
+    >>> import numpy as np
+    >>> from toqito.channel_ops import kraus_to_channel
+    >>> kraus_1 = np.array([[1, 0], [0, 0]])
+    >>> kraus_2 = np.array([[0, 1], [0, 0]])
+    >>> kraus_list = [(kraus_1, kraus_1), (kraus_2, kraus_2)]
+    >>> kraus_to_channel(kraus_list)
+    array([[1, 0, 0, 1],
+           [0, 0, 0, 0],
+           [0, 0, 0, 0],
+           [0, 0, 0, 0]])
+
+    See Also
+    ========
+    func:`.choi_to_kraus`, func:`.kraus_to_choi`
```

Comment:
@Shivansh20128 something has gone wrong with the formatting for this. I was trying to make these clickable similar to what I did in #1091 

![image](https://github.com/user-attachments/assets/95ff2730-39b9-4d29-b204-5bfcfad68867)